tools/create-genesis/src/miner.rs

```diff
@@ -0,0 +1,51 @@
+use crate::common::{get_temp_state, secret_2_address, string_2_bytes, string_2_u256};
+use crate::genesis::Account;
+use core_executor::engines::NullEngine;
+use core_executor::executive::contract_address;
+use core_executor::libexecutor::sys_config::BlockSysConfig;
+use core_executor::types::transaction::Transaction;
+use evm::cita_types::U256;
+use evm::env_info::EnvInfo;
+use libproto::blockchain::Transaction as ProtoTransaction;
+
+pub struct Miner;
```

Comment:
Why needs a struct?
Is it better than just define a pub function mine?

Reply:
OOP, It's more readable in this way.